doc/Developing/os/Health-Information.md

```diff
@@ -150,6 +150,47 @@ well as pre-fetched data. The index ($index) and the sub_indexes (in
 case the oid is indexed multiple times) are also available: if
 $index="1.20", then $subindex0="1" and $subindex1="20".
 
+To fetch data not avaialble to your sensor you can use `additional_oids`.
+
+> [!IMPORTANT] 
```

Comment:
Can you replace `[!IMPORTANT]` with `!!! note` (or danger if you want it red).

Reply:
Went with caution

Reply:
It needs to `!!!` not `[!]`

Reply:
@martinberg Can you just trust that we know what we're talking about. Please update `>` to `!!! note`